frontend/agents/gpt-4_python_compute/generate/computations.py

```diff
@@ -109,7 +109,7 @@ def compute(user_prompt, model_version, conversation_history, apiKey):
         apiKey = data["apiKey"]
 
         # Call the compute function and get the result
-        result = compute(user_prompt, 'gpt-4', conversation_history, apiKey)
+        result = compute(user_prompt, 'gpt-4o', conversation_history, apiKey)
```

Comment:
There's no need to update the langchain version for this? We also might want to upgrade it for the view agent as well.

Reply:
Just changes the model in the api call. Added to view